manager/sinks.go

```diff
@@ -22,9 +22,9 @@ import (
 	sink_api "github.com/GoogleCloudPlatform/heapster/sinks/api/v1"
 )
 
-func newSinks() ([]sink_api.ExternalSink, error) {
+func newSinks(sinkUris []string) ([]sink_api.ExternalSink, error) {
```

Comment:
I'm OK with doing in another PR. I don't think we should change the flags though. Writing JSON to flags is not the nicest...